rero_ils/modules/patrons/api.py

```diff
@@ -249,13 +249,6 @@ def _get_user_by_user_id(cls, user_id):
         """
         return _datastore.find_user(id=user_id)
 
-    @property
-    def profile_url(self):
-        """Get the link to the RERO_ILS patron profile URL."""
-        view_code = self.get_organisation().get('code')
-        base_url = current_app.config.get('RERO_ILS_APP_URL')
-        return f'{base_url}/{view_code}/patrons/profile'
-
```

Comment:
Why ? 
This is used into `rero_ils.modules.patrons.dumpers:PatronNotificationDumper` and include into many notification template.

Reply:
This property is defined twice, see:
https://github.com/rero/rero-ils/blob/US_request_holdings/rero_ils/modules/patrons/api.py#L253
https://github.com/rero/rero-ils/blob/US_request_holdings/rero_ils/modules/patrons/api.py#L267